test/test-util.ts

```diff
@@ -141,9 +140,12 @@ describe('util', () => {
       include: ['c'],
     };
     const myMap = new Map();
-    myMap.set('/some/fake/directory/tsconfig.json', FAKE_CONFIG1);
-    myMap.set('/some/fake/directory/foo/FAKE_CONFIG2', FAKE_CONFIG2);
-    myMap.set('/some/fake/directory/foo/bar/FAKE_CONFIG3', FAKE_CONFIG3);
+    myMap.set(PATH_TO_TSCONFIG, FAKE_CONFIG1);
+    myMap.set(path.resolve(FAKE_DIRECTORY, './foo/FAKE_CONFIG2'), FAKE_CONFIG2);
```

Comment:
using path.resolve here on win32 will flip all the slashes. is this a problem?
see 
`path.win32.resolve('/a/b/c','./foo/boo.js')` vs `path.posix.resolve('/a/b/c','./foo/boo.js')`

Reply:
i see. we pass all of the path strings through resolve in some way before using them :+1: